### PR TITLE
zh-CN: site/zh-cn/beta/tutorials/load_data/csv.ipynb

### PR DESCRIPTION
**Pull request by surfingbyte**
I have translated site/zh-cn/beta/tutorials/load_data/csv.ipynb with Chinese, please review it.

### CODE REVIEW

File: site/zh-cn/beta/tutorials/load_data/csv.ipynb [hunk 1]
```diff
@@ -0,0 +1,694 @@
...
+        "    <a target=\"_blank\" href=\"https://www.tensorflow.org/beta/tutorials/load_data/text\"><img src=\"https://www.tensorflow.org/images/tf_logo_32px.png\" />View on TensorFlow.org</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/tensorflow/docs/blob/master/site/en/r2/tutorials/load_data/csv.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Run in Google Colab</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs/blob/master/site/en/r2/tutorials/load_data/csv.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />View source on GitHub</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a href=\"https://storage.googleapis.com/tensorflow_docs/site/en/r2/tutorials/load_data/csv.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\" />Download notebook</a>\n",
```
Comment (Mr-Linus) [inline site/zh-cn/beta/tutorials/load_data/csv.ipynb]: 从 70 - 82 行，请根据 TF 汉化社区格式标准修改。https://docs.google.com/document/d/1v7kwEhvtbbI4S4gulxLJDDN78gv0ZlDJ0cN92BUPrm0

Comment (surfingbyte) [reply site/zh-cn/beta/tutorials/load_data/csv.ipynb]: done

File: site/zh-cn/beta/tutorials/load_data/csv.ipynb [hunk 2]
```diff
@@ -0,0 +1,694 @@
...
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://github.com/tensorflow/docs/blob/master/site/zh-cn/beta/tutorials/load_data/csv.ipynb\"><img src=\"https://tensorflow.google.cn/images/GitHub-Mark-32px.png\" />在 Github 上查看源代码</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a href=\"https://storage.googleapis.com/tensorflow_docs/site/zh-cn/beta/tutorials/load_data/csv.ipynb\"><img src=\"https://tensorflow.google.cn/images/download_logo_32px.png\" />下载此 notebook</a>\n",
+        "  </td>\n",
+        "</table>"
+      ]
+    },
+    {
```
Comment (lamberta) [inline site/zh-cn/beta/tutorials/load_data/csv.ipynb]: Add disclaimer for community translations. This is something we need to add on all community translated content. See [this notebook](https://github.com/tensorflow/docs/blob/master/site/zh-cn/beta/tutorials/quickstart/beginner.ipynb) as an example.

**Comment by Mr-Linus** [comment]
@lamberta 👌

**Pull request merged**